Add GetGrenadeHoldTime function

Into-The-Rough opened the pull request:
Add GetGrenadeHoldTime to calculate time player is holding a grenade before throwing.

lStewieAl commented on JG/functions/fn_gameplay.h:
```diff
@@ -1484,5 +1486,10 @@ bool Cmd_PathToRef_Execute(COMMAND_ARGS) {
+}
+
+bool Cmd_GrenadeTimeHeld_Execute(COMMAND_ARGS) {
+	*result = *(float*)((UInt32)g_thePlayer + 0xE50);
```
Please use a field, don't just offset from the player.

Into-The-Rough commented:
@lStewieAl sorry for slow response, I basically pushed a change that was wrong. The original I copied from SUP, and was being lazy.

I had a chat with @Wadela and I believe this is the right approach. It works in-game I have verified.

Into-The-Rough committed:
Add GetGrenadeHoldTime function

Fix PlayerCharacter struct alignment by placing timeGrenadeHeld field
at offset 0x730. Implements command to return grenade hold time with
proper NVSE naming convention.

WallSoGB committed:
Merge branch 'dev' into add-grenade-time-held
